neqo-transport/src/send_stream.rs

```diff
@@ -740,6 +747,12 @@ impl SendStreams {
         self.0.retain(|_, stream| !stream.is_terminal())
     }
 
+    pub(crate) fn retry(&mut self) {
+        for (_, stream) in self {
```

Comment:
yes SendStreams implements &mut IntoIterator  but it might be clearer to use tuple notation inside SendStreams. e.g. `for stream in self.0.values()` instead here. That's what other methods are doing.